src/pages/CircleAdminPage/AdminIntegrations.tsx

```diff
@@ -13,7 +13,7 @@ import {
   ParcelIcon,
 } from 'icons';
 import { paths } from 'routes/paths';
-import { Flex, Button } from 'ui';
+import { Flex, Button, Text } from 'ui';
 
 const useStyles = makeStyles(theme => ({
```

Comment:
Please remove the use of makeStyles from this file

Reply:
I will do it but tried to do minimum changes here because there is a separate issue for Integration section